src/libraries/Microsoft.PowerFx.Core/Public/ExpressionError.cs

```diff
@@ -57,12 +57,15 @@ public string Message
 
         public ErrorSeverity Severity { get; set; } = ErrorSeverity.Severe;
 
-        public string MessageKey { get; set; }
+        public ErrorResourceKey ResourceKey { get; set; }
```

Comment:
>set [](http://example.com/codeflow?start=51&length=3)

API compat is showing we removed the setter, which is technically breaking. I think that's ok.